### PR TITLE
Chore: remove IP restrictions

### PR DESCRIPTION
**Pull request by angela-tran**
Talked with @thekaveman, and these are not a requirement for the initial release. We will revisit this in the next release.

### CODE REVIEW

**Comment by afeld** [comment]
Why don't we just clear the variable? No code change needed.

**Review by afeld — DISMISSED** [review]

File: terraform/app_service.tf [hunk 1]
```diff
@@ -19,13 +19,7 @@ resource "azurerm_linux_web_app" "main" {
...
 
   site_config {
-    ftps_state = "Disabled"
-    dynamic "ip_restriction" {
-      for_each = var.IP_ADDRESS_WHITELIST
-      content {
-        ip_address = ip_restriction.value
-      }
-    }
+    ftps_state             = "Disabled"
```
Comment (afeld) [inline terraform/app_service.tf:22]: In that case:

```suggestion
    # required to remove the restrictions
    ip_restriction         = []
    ftps_state             = "Disabled"
```

Comment (afeld) [reply terraform/app_service.tf:22]: Wait, nevermind, you already said this.

Comment (angela-tran) [reply terraform/app_service.tf:22]: Actually I agree with you - I think being explicit like this is better. Also, it turns out the pipeline put back the IP restriction in a [recent build](https://dev.azure.com/mstransit/courtesy-cards/_build/results?buildId=88&view=logs&j=bc150913-bfac-577d-ab97-4eb22da69064&t=9e090432-7346-5314-b3f7-b310ded2d897) 🙃

**Comment by afeld** [comment]
I merged another branch and the manual change got undone, so to get the dev site up again, I changed the variable in the pipeline, manually triggered a deploy of `dev`, and manually removed the IP restriction in the Portal. Wanted to make sure UAT wasn't blocked, hope that was ok.

I don't think it's necessary to merge this since we intend to undo it soon, but your call.

**Review by afeld — APPROVED** [review]
Your call about whether to merge or not. Good that it's expliclt, but shouldn't change anything.

**Comment by angela-tran** [comment]
Since the variable has been cleared, we don't expect to accidentally put the restriction back. Going to just close this PR and leave the harmless dynamic block in since we'll come back to it in the next release.

**Pull request closed without merging**